static/sass/styles.scss

```diff
@@ -678,5 +685,65 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
 }
 
 .is-ticked.is-play-button::before {
-  @include vf-icon-video-play($color-mid-dark);
+  @include vf-icon-video-play($color-mid-dark); 
+}
+
+// XXX: pad content so it doesn't get to close to text in the adjacent column. Idea borrowed from the guardian. Propose this to vanilla.
+.u-pad--right {
+  padding-right: $sph--x-large;
+}
+
+.u-align-social-icons-to-text {
```

Comment:
Again the same comment here. This should be part of the molecule style not another utility which will never be used right